giskard/slicing/tree_slicer.py

```diff
@@ -59,6 +60,7 @@ def find_slices(self, features, target=None):
         if len(data) < min_samples:
             return []
 
+        mode = None
```

Comment:
this is ugly, just pass the class name to the telemetry

Reply:
But that wouldn't tell us if it's classification or regression and I think that's important to know (I agree it's ugly though)

Reply:
Fixed